repository/src/main/resources/alfresco/repository.properties

```diff
@@ -168,6 +168,7 @@ system.auditableData.ACLs=${system.auditableData.preserve}
 # Properties to control read permission evaluation for acegi
 system.readpermissions.optimise=true
 system.readpermissions.bulkfetchsize=1000
+system.acl.maxPermissionCheckEnabled=false

```

Comment:
This should not be here, but at line 156, where the related properties are defined. Properties of the same context should be organised together.

Reply:
Correct, I will move it up in the right place.